### PR TITLE
Google Plus sing in functionality

### PR DESCRIPTION
**Pull request by Enanodr**
## Summary
- Gmail sing in functionality
- All the app ids where created from the acount diego.raffo@wolox.com.ar
- Google+ API and Contacts API had to be enabled from Google developers console

## TrelloCard
https://trello.com/c/ZPJiDqYU

## Sign in using Gmail
image

### CODE REVIEW

File: blog/app/models/user.rb [hunk 1]
```diff
@@ -1,6 +1,25 @@
 class User < ActiveRecord::Base
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :omniauthable, :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable
+  devise :omniauthable, :database_authenticatable, :registerable,:recoverable, :rememberable, :trackable
+
+  def google_oauth2
+    identities.where( :provider => "google_oauth2" ).first
```
Comment (epintos) [inline blog/app/models/user.rb]: find_by_provider('google_oauth2')

**Comment by epintos** [comment]
You should run rubocop here also

**Comment by Enanodr** [comment]
Rubocop was developed in a separated branch, its currently on master. 
Run rubocop directly from master after merged?

**Comment by epintos** [comment]
Fast forward and :+1:

**Comment by Enanodr** [comment]
Merge manually

**Pull request closed without merging**